configure.ac

```diff
@@ -9,38 +9,61 @@ AC_CONFIG_HEADER(config.h)
 
 AC_CANONICAL_HOST
 
+AC_SUBST(ccache_os, unknown)
 case $host in
     *mingw32* | *mingw64* | *cygwin* | *wince* | *mingwce*)
-        windows_os=yes
+        ccache_os=mingw
```

Comment:
The good question is: what is the difference between MinGW and MSYS2 ?
I found at least one:
- MSYS2: You should link with `Psapi.lib`
- MINGW: You should link with `-lpsapi`  (sorry for the mixed case typo)